### DISCUSSION TITLE
CHAOSS Asia-Pacific Community Meeting 6/1/22

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#70] [1️⃣ 新鲜事]
可以简单关注下：https://www.youtube.com/watch?v=egCbgWqeVI4
- Metric model 进展
- 新加入的高校老师同学（南京大学）
- 期盼 CHAOSS China 例会的重启
- .等等

**Comment by xiaoya-yaya** [comment]
原来 Elizabeth 是以前修的是组织行为学，这专业也太适合做社区经理了哈哈

**Reply by will-ww** [reply]
所以，很多时候，把这些细节挖出来，还是挺有意思的，也有不少启发~
